cmd/ethereum_rust/ethereum_rust.rs

```
@@ -70,10 +70,11 @@ async fn main() {
     let tcp_socket_addr =
         parse_socket_addr(tcp_addr, tcp_port).expect("Failed to parse addr and port");
 
-    let _genesis = read_genesis_file(genesis_file_path);
+    let mut store = Store::new("storage.db", EngineType::InMemory).expect("Failed to create Store");
+    let genesis = read_genesis_file(genesis_file_path);
+    let _ = store.add_initial_state(genesis);
```

Could we use an `expect` here instead of ignoring the error?